app/views/admin/bulk_uploads/set_titles.html.erb

```diff
@@ -1,34 +1,65 @@
-<% page_title "Bulk upload for: #{@edition.title}" %>
+<% content_for :page_title, "Bulk upload for: #{@edition.title}" %>
+<% content_for :title, "Edit file attachment" %>
+<% content_for :context, "Attachments for publication" %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: admin_edition_attachments_path(@edition)
+  } %>
+<% end %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>Set titles for your attachments</h1>
-    <%= form_for @bulk_upload, url: admin_edition_bulk_uploads_path(@edition) do |form| %>
-      <%= form.errors %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-!-margin-bottom-8">
+      <%= form_for @bulk_upload, url: admin_edition_bulk_uploads_path(@edition) do |form| %>
+        <% content_for :error_summary, render('shared/error_summary', object: @bulk_upload, parent_class: "bulk_upload", class_name: @bulk_upload.class.name.demodulize.underscore.humanize.downcase) %>
```

Comment:
The validation is slightly odd on this form.

It looks like it doesn't actually show error messages for each invalid form field – just a generic "fill in missing fields" prompt.

![Error message when adding attachment details](https://user-images.githubusercontent.com/7735945/195338409-6ed91ae3-e303-4f9a-bbbc-3d1a036c4af9.png)

We're using the 'error summary' component here, meaning the error messages should be clickable and refer directly to the invalid input fields. But in this case, clicking the error message does nothing – it adds `#bulk_upload_base` to the URL, but there isn't an element present on the page with that ID.

This feels wrong, but I'm not sure what the best course of action is here. Do we spend time improving validation, or accept that it's functionally the same as what exists in Bootstrap and move on? That feels like a product decision to me.

Reply:
Yes, I discussed this with Carlos at the time and we decided to go with it as it is for now (see comments on [card](https://trello.com/c/UCQ6nWML/613-move-bulk-upload-attachments-page-to-the-govuk-design-system)). As far as I am aware the multiple files aspect of this causes some problems with the error messaging as it exists. I also started writing a new ticket to cover this in future. I'll make sure that exists before moving on from this ticket.